src/LuaTransformer.ts

```diff
@@ -911,8 +913,53 @@ export class LuaTransformer {
         );
     }
 
-    public transformExpressionStatement(statement: ts.ExpressionStatement): StatementVisitResult {
-        return tstl.createExpressionStatement(this.transformExpression(statement.expression));
+    public transformExpressionStatement(expression: ts.Expression): StatementVisitResult {
```

Comment:
Maybe overload this so we can pass both `ts.Expression` and `ts.ExpressionStatement`